app/config/AppConfig.scala

```diff
@@ -35,40 +35,42 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   lazy val customsFinancialsApi: String = servicesConfig.baseUrl("customs-financials-api") +
     config.getOptional[String]("customs-financials-api.context").getOrElse("/customs-financials-api")
 
-  lazy val customsDataStore: String = s"${servicesConfig.baseUrl("customs-data-store")}${
-    config.getOptional[String](
-      "microservice.services.customs-data-store.context").getOrElse("/customs-data-store")
-  }"
+  private lazy val customsDataStoreBaseUrl    = servicesConfig.baseUrl("customs-data-store")
+  private lazy val customsDataStoreContextUrl =
+    config.getOptional[String]("microservice.services.customs-data-store.context").getOrElse("/customs-data-store")
+  lazy val customsDataStore: String           =
+    s"$customsDataStoreBaseUrl$customsDataStoreContextUrl"
 
-  lazy val emailFrontendUrl: String = s"${servicesConfig.baseUrl("customs-email-frontend")}${
+  private lazy val emailFrontendBaseUrl    = servicesConfig.baseUrl("customs-email-frontend")
+  private lazy val emailFrontendContextUrl =
     config.getOptional[String]("customs-email-frontend.context").getOrElse("/manage-email-cds")
-  }${
-    config.get[String]("microservice.services.customs-email-frontend.url")
-  }"
-
-  lazy val appName: String = config.get[String]("appName")
-  lazy val loginUrl: String = config.get[String]("urls.login")
-  lazy val loginContinueUrl: String = config.get[String]("urls.loginContinue")
-  lazy val signOutUrl: String = config.get[String]("urls.signOut")
-  lazy val cashAccountTopUpGuidanceUrl: String = config.get[String]("urls.cashAccountTopUpGuidanceUrl")
+  private lazy val emailFrontendFinalUrl   = config.get[String]("microservice.services.customs-email-frontend.url")
+  lazy val emailFrontendUrl: String        =
```

Comment:
Better to give a line break before this line to make it more readable and to differentiate from private vals